ulwgl_run.py

```diff
@@ -327,8 +328,16 @@ def main() -> None:  # noqa: D103
         os.environ[key] = val
 
     build_command(env, command, opts)
-    subprocess.run(command)
+    proc = subprocess.run(command)
+    return proc.returncode
 
 
 if __name__ == "__main__":
-    main()
+    try:
+        ret = main()
```

Comment:
Prefer:

> sys.exit(main())

Reply:
I do not really like it this way, without good reason though. Ideally I would assign `ret` and exit in a `finally` block but since I do not really have a good reason for it, outside of aesthetics I will do it the suggested way